pb_plugins/dcsdkgen/type_info.py

```diff
@@ -2,6 +2,24 @@
 from os import environ
 
 
+class TypeInfoFactory:
+
+    def __init__(self):
+        self._conversion_dict = self.__load_conversions_dict()
+
+    def create(self, field):
+        return TypeInfo(field, self._conversion_dict)
+
+    def __load_conversions_dict(self):
+        try:
+            _conversion_dict_path = environ.get(
```

Comment:
Could be more readable when using an formatted String (`f"this is a text with {variable_name_or_function}"`